app/uk/gov/hmrc/tradergoodsprofilesrouter/controllers/action/ValidationRules.scala

```diff
@@ -114,6 +114,8 @@ object ValidationRules {
 
   private val actorIdPattern: Regex = raw"[A-Z]{2}\d{12,15}".r
   private val comcodePattern: Regex = raw".{6}(.{2}(.{2})?)?".r
+  private val niphlsPattern: Regex  = """^\d{4}$|^\d{5}$|^\d{6}$|^[A-Za-z]\d{5}$|^[A-Za-z]\d{6}$""".r
+//  private val niphlsPattern: Regex  = raw"^\d{4}$|^\d{5}$|^\d{6}$|^[A-Za-z]\d{5}$|^[A-Za-z]\d{6}".r
```

Comment:
can we remove the commented code if its not going to be used ever?

Reply:
updated